doc/config_manager.rst

```diff
@@ -86,6 +86,12 @@ Examples
 ``dnf config-manager --set-enabled <repoid>``
     Enable repository identified by <repoid> and make the change permanent.
 
+``dnf config-manager --disable <repoid1> <repoid2>``
+    Disable repositories identified by <repoid1> and <repoid2>
+
+``dnf config-manager --disable <repoid1>,<repoid2>``
+    Disable repositories identified by <repoid1> and <repoid2>
```

Comment:
This is maybe just a matter of preference, but can you rather use ``--set-disabled``? We added the ``--enable`` and ``--disable`` options only for compatibility reasons and these options are not listed in the config-manager help section (only in the general section). Also, one of the other examples above uses the ``--set-*`` variant and it would be better to stay consistent and promote just one of the variants.